cloudbuild_bazel.sh

```diff
@@ -4,162 +4,4 @@ set -o errexit
-bazel run //nodejs:nodejs14_arm64_debian9
-bazel run //nodejs:nodejs14_debug_arm64_debian9
-bazel run //nodejs:nodejs14_arm64_debian10
-bazel run //nodejs:nodejs14_debug_arm64_debian10
-
-bazel run --host_force_python=PY2 //experimental/dotnet:dotnet_debian9
-bazel run --host_force_python=PY2 //experimental/dotnet:dotnet_debian10
-bazel run --host_force_python=PY2 //experimental/dotnet:dotnet_core_aspnet_debian9
-bazel run --host_force_python=PY2 //experimental/dotnet:dotnet_core_runtime_debian9
-bazel run --host_force_python=PY2 //experimental/dotnet:dotnet_core_sdk_debian9
-
-bazel run --host_force_python=PY2 //experimental/dotnet:dotnet_debug_debian9
-bazel run --host_force_python=PY2 //experimental/dotnet:dotnet_debug_debian10
-bazel run --host_force_python=PY2 //experimental/dotnet:dotnet_core_aspnet_debug_debian9
-bazel run --host_force_python=PY2 //experimental/dotnet:dotnet_core_runtime_debug_debian9
-bazel run --host_force_python=PY2 //experimental/dotnet:dotnet_core_sdk_debug_debian9
-
-    bazel run --host_force_python=PY2 //java:java_base_root_debian9
-    bazel run --host_force_python=PY2 //java:java_base_nonroot_debian9
-    bazel run --host_force_python=PY2 //java:java_base_debug_root_debian9
-    bazel run --host_force_python=PY2 //java:java_base_debug_nonroot_debian9
-    bazel run --host_force_python=PY2 //java:java_base_root_debian10
-    bazel run --host_force_python=PY2 //java:java_base_nonroot_debian10
-    bazel run --host_force_python=PY2 //java:java_base_debug_root_debian10
-    bazel run --host_force_python=PY2 //java:java_base_debug_nonroot_debian10
-    bazel run --host_force_python=PY2 //java:java8_root_debian9
-    bazel run --host_force_python=PY2 //java:java8_nonroot_debian9
-    bazel run --host_force_python=PY2 //java:java8_debug_root_debian9
-    bazel run --host_force_python=PY2 //java:java8_debug_nonroot_debian9
-    bazel run --host_force_python=PY2 //java:java11_root_debian9
-    bazel run --host_force_python=PY2 //java:java11_nonroot_debian9
-    bazel run --host_force_python=PY2 //java:java11_debug_root_debian9
-    bazel run --host_force_python=PY2 //java:java11_debug_nonroot_debian9
-    bazel run --host_force_python=PY2 //java:java11_root_debian10
-    bazel run --host_force_python=PY2 //java:java11_nonroot_debian10
-    bazel run --host_force_python=PY2 //java:java11_debug_root_debian10
-    bazel run --host_force_python=PY2 //java:java11_debug_nonroot_debian10
-
-    bazel run --host_force_python=PY2 //java/jetty:jetty_java8_debian9
-    bazel run --host_force_python=PY2 //java/jetty:jetty_java8_debug_debian9
-    bazel run --host_force_python=PY2 //java/jetty:jetty_java11_debian9
-    bazel run --host_force_python=PY2 //java/jetty:jetty_java11_debug_debian9
-    bazel run --host_force_python=PY2 //java/jetty:jetty_java11_debian10
-    bazel run --host_force_python=PY2 //java/jetty:jetty_java11_debug_debian10
+bazel run --host_force_python=PY2 //:publish
```

Comment:
Do need this script? What is we embed these two commands into `cloudbuild.yaml`? Or, was it that you will eventually remove `cloudbuild.yaml`? In any case, rename the file to `build_and_publish.sh`? And is it safe to run `cloudbuild_bazel.sh` locally? (I guess it will fail because `{PROJECT_ID}` is undefined?) Maybe now it's worth adding some comments into this script?

Reply:
Nope, we could just move cloudbuild_bazel.sh into cloudbuild.yaml now. 👍 

When/if rules_docker supports manifest lists we can assemble and publish those using Bazel too and remove that script 🙏